torch/testing/_internal/common_modules.py

```diff
@@ -3265,7 +3267,9 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
     ModuleInfo(torch.nn.Tanhshrink,
                module_inputs_func=module_inputs_torch_nn_Tanhshrink,
                skips=(
-                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),)
+                   DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),
+                   # Fails with channels last test on MPS backend
+                   DecorateInfo(unittest.expectedFailure, "TestModule", "test_memory_format", device_type='mps'),)
```

Comment:
Same as above?

Reply:
Same as above. It's not a regression and pre-existed, The skip is introduced in another PR, so I'll remove this change from my PR:
https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_modules.py#L3409